index.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title>Weather App</title>
+	<link rel="stylesheet" href="src/style.css">
+	<link href="https://fonts.googleapis.com/css2?family=PT+Serif&display=swap" rel="stylesheet">
+	<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css"
+		integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk" crossorigin="anonymous">
+	<script src="https://kit.fontawesome.com/200b8902d2.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
+	<div class="boxed">
+		<div class="container">
+			<div class="topnav">
+				<a class="active" href="#home">Search</a>
+				<input type="text" placeholder="Enter Current Location">
+			</div>
+			<button type="button" class="btn btn-primary">Search for more locations here</button>
+			<h1>
+				San Jose
+			</h1>
+			<h2>
+				7/18/20 8pm
+			</h2>
+			<h3>
+				<i class="fas fa-cloud"></i>
+			</h3>
+			<h4>
+				75° F
+			</h4>
```

Comment:
This section could be a bit better organized and I think it would really help you with laying it all out. Remember - siblings in an HTML document are laid out by default on top of each other (i.e. `"topnav"` will be on top of `button` which will be on top of `h1` and so on...). It would make more sense to group these how you want them laid out in the final product. You are on the right track with the `"topnav"` div. The search button and the search input are one unit and can be logically grouped together inside of a div. And that unit is then laid out on top the button. However, there is still some more logical grouping that we can do. For example, in the final product we want "San Jose" to be on top of "7/18/20 8pm" which is supposed to be on top of "75° F". And we want those three things to be next to the icon. And we want all of that to be to the left of the search controls. The goal is to think about it in terms of units that can be placed either next to or on top of each other. The three headings are one unit because all three of them are placed on top of each and need to be positioned all together. Looking at this right now I would break it up like this:

```html
<div class="header-display">
	<!-- This whole section is one unit because it all needs to be positioned to the left of the location info -->
	<div class="search-controls">
		<div class="topnav">
			<a class="active" href="#home">Search</a>
			<input type="text" placeholder="Enter Current Location">
		</div>
		<button type="button" class="btn btn-primary">Search for more locations here</button>
	</div>

	<!-- This whole section is one unit because it all needs to be positioned to the right of the search controls -->
	<div class="location-info">
 		<!-- These three headers make up one unit because all three go on top of each other -->
		<div class="text">
			<h1>San Jose</h1>
			<h2>7/18/20 8pm</h2>
			<h3>75° F</h3>
		</div>
		<!-- This icon is one unit on it's own which needs to be positioned next to the text -->
		<div class="icon">
			<i class="fas fa-cloud"></i>
		</div>
	</div>
<div>
```

It doesn't have to be this exactly, but if you have something like this, it becomes a *lot* easier to lay it out with proper spacing with flex.

Reply:
As a general rule of thumb, if you have some elements that need to be on top of each other and some elements that need to be next to each other in the same div, then you should probably break them down into more divs.